internal/codegen/stencil_test.go

```diff
@@ -95,14 +91,11 @@ func TestModuleHookRender(t *testing.T) {
 	assert.Equal(t, tpls[1].Files[0].String(), "a", "expected Render() to return correct output")
 }
 
-func ExampleStencil_PostRun() {
-	fs := memfs.New()
+func TestExampleStencil_PostRun(t *testing.T) {
```

Comment:
suggestion: I don't think that's how Examples are supposed to be defined, according to the docs. In this case you may just want to revert the changes to this function and leave the `memfs` code the way it was.